simulation/simulation_interface/proto/openscenario_msgs.proto

```diff
@@ -6,125 +6,114 @@ package openscenario_msgs;
+  double max_speed = 1;        // Max speed of the entity.
+  double max_acceleration = 2; // Max acceleration of the entity.
+  double max_deceleration = 3; // Max deceleration of the entity.
 }
 
 /**
  * Protobuf definition of openscenario_msgs/msg/Axle type in ROS2.
-**/
-message Axle
-{
-    double max_steering = 1; // Max steering of the entity axle.
-    double wheel_diameter = 2; // Wheel diameter of the entity axle.
-    double track_width = 3; // Track width of the entity axle.
-    double position_x = 4; // Position x of the entity axle.(longitudinal)
-    double position_z = 5; // Position z of the entity axle.(lateral)
+ **/
+message Axle {
+  double max_steering = 1;   // Max steering of the entity axle.
+  double wheel_diameter = 2; // Wheel diameter of the entity axle.
+  double track_width = 3;    // Track width of the entity axle.
+  double position_x = 4;     // Position x of the entity axle.(longitudinal)
+  double position_z = 5;     // Position z of the entity axle.(lateral)
 }
 
 /**
  * Protobuf definition of openscenario_msgs/msg/Axles type in ROS2.
-**/
-message Axles
-{
-    Axle front_axle = 1; // Parameters of the front axle of the entity.
-    Axle rear_axle = 2; // Parameters of rear axle of the entity.
+ **/
+message Axles {
+  Axle front_axle = 1; // Parameters of the front axle of the entity.
+  Axle rear_axle = 2;  // Parameters of rear axle of the entity.
 }
 
 /**
  * Protobuf definition of openscenario_msgs/msg/Property type in ROS2.
-**/
-message Property
-{
-}
+ **/
+message Property {}
 
 /**
  * Protobuf definition of openscenario_msgs/msg/VehicleParameters type in ROS2.
-**/
-message VehicleParameters
-{
-    string name = 1; // Name of the vehicle entity
-    string vehicle_category = 2; // Category of the vehicle entity.
-    Performance performance = 3; // Performance parameter of vehicle entity.
-    BoundingBox bounding_box = 4; // Bounding box of the vehicle entity.
-    Axles axles = 5; // Axles of the vehicle entity.
-    Property property = 6; // Other parameters of the vehicle entity.
+ **/
+message VehicleParameters {
+  string name = 1;              // Name of the vehicle entity
+  string vehicle_category = 2;  // Category of the vehicle entity.
+  Performance performance = 3;  // Performance parameter of vehicle entity.
+  BoundingBox bounding_box = 4; // Bounding box of the vehicle entity.
+  Axles axles = 5;              // Axles of the vehicle entity.
+  Property property = 6;        // Other parameters of the vehicle entity.
 }
 
 /**
- * Protobuf definition of openscenario_msgs/msg/PedestrianParameters type in ROS2.
-**/
-message PedestrianParameters
-{
-    string name = 1; // Name of the pedestrian entity.
-    string pedestrian_category = 2; // Category of pedestrian entity.
-    BoundingBox bounding_box = 3; // Bounding box of the pedestrian entity.
+ * Protobuf definition of openscenario_msgs/msg/PedestrianParameters type in
+ *ROS2.
+ **/
+message PedestrianParameters {
```

Comment:
Malformed indentation

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/357/commits/082e52f817ed4a829eeddfdd9148cd0455d8661b

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/357/commits/d05e45028e960780dfb581d3dd15573f3f18f803